### PR TITLE
Fixes #8096: 

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8096

### CODE REVIEW

File: initial-promises/node-server/server-roles/1.0/password-check.cf [hunk 1]
```diff
@@ -55,6 +55,15 @@ bundle agent root_password_check_disclaimer
 
 }
 
+bundle agent root_password_security
+{
+  files:
+    # ensure password access is forbidden
+    root_server::
+      "${managed_files}"
```
Comment (fanf) [inline initial-promises/node-server/server-roles/1.0/password-check.cf]: managed_files is at least missing rudder-passwords.conf

Comment (ncharles) [inline initial-promises/node-server/server-roles/1.0/password-check.cf]: p.managed_files

File: techniques/system/server-roles/1.0/password-check.st [hunk 2]
```diff
@@ -55,6 +55,15 @@ bundle agent root_password_check_disclaimer
 
 }
 
+bundle agent root_password_security
+{
+  files:
+    # ensure password access is forbidden
+    root_server::
+      "${managed_files}"
```
Comment (ncharles) [inline techniques/system/server-roles/1.0/password-check.st]: same here

Comment (ncharles) [inline techniques/system/server-roles/1.0/password-check.st]: it also needs at least any .rpmnew files that might have been created by package (and probably dpkg-new ?)
and also rudder-users.xml, rudder-slapd.conf and rudder_networks.*

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

File: initial-promises/node-server/server-roles/1.0/password-check.cf [hunk 3]
```diff
@@ -55,6 +55,17 @@ bundle agent root_password_check_disclaimer
 
 }
 
+bundle agent root_password_security
+{
+  vars:
+    "secure_files" slist => { "@{p.managed_files}", "/opt/rudder/etc/rudder-passwords.conf", "/opt/rudder/etc/rudder-users.xml", "/opt/rudder/etc/rudder-slapd.conf", "/opt/rudder/etc/rudder-networks.conf" };
```
Comment (ncharles) [inline initial-promises/node-server/server-roles/1.0/password-check.cf]: and /opt/rudder/etc/rudder-networks-24.conf

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**